docs/notebooks/nmf_benchmark.ipynb

```diff
@@ -0,0 +1,1159 @@
+{
```

Comment:
- Re-run notebook (I see KeyboardInterrupt in some training)
- More description (not only headers), links to referred notebooks (sklearn), what we do here and why